pkg/plugins/golang/deploy-image/v1alpha1/scaffolds/api.go

```diff
@@ -111,6 +111,12 @@ func (s *apiScaffolder) Scaffold() error {
 		return fmt.Errorf("error updating config/samples: %v", err)
 	}
```

Comment:
@NikhilSharmaWe see that is failing because now it will try to use/install the kubebuilder tools to run the make test on the e2e. We need to change the e2e test:

Replace make docker-build IMG=<image> with 
` docker build -t <image value> . `